dlibs/dlib_reactionroles.py

```diff
@@ -46,17 +46,16 @@ async def on_raw_reaction_add(payload, **kargs):
                     pass
 
 
-async def on_raw_reaction_remove(payload, **kargs):
+async def on_raw_reaction_remove(payload: discord.RawReactionActionEvent, **kargs: Any) -> None:
 
     if not payload.guild_id: return
 
     inc_statistics([payload.guild_id, "on-raw-reaction-remove", kargs["kernel_ramfs"]])
 
     client = kargs["client"]
-    rrconf = load_message_config(payload.guild_id, kargs["ramfs"], datatypes=reactionrole_types)["reaction-role-data"]
+    rrconf: Any = load_message_config(payload.guild_id, kargs["ramfs"], datatypes=reactionrole_types)["reaction-role-data"]
```

Comment:
line 36 this was Dict what is going on here?

Reply:
line 36 used to be Any this was before the JSON cache type was made (that returns a pure pyobj)